text/1001-edgedb-install.rst

```diff
@@ -0,0 +1,298 @@
+the current platform and install it using the system's package manager
+if one is available, or download and unpack a generic package for the
+OS/arch, or use Docker if avalable (and requested).
+
+
+edgedb server uninstall
+=======================
+
+Uninstalls the specified version of EdgeDB.
+
+Synopsis
+--------
+
+``edgedb server uninstall [options]``
+
+If there are multiple versions installed, either ``--all`` or
+``--version`` or ``--unused`` is required.
+
+Options
+-------
+
+``--version=<ver>``
+  Specifies the version to uninstall.  The specified server version must
+  not be currently running.
+
+``--all``
+  Uninstalls all versions of EdgeDB.
+
+``--unused``
+  Uninstalls all versions of EdgeDB that are not used in any instance.
+
+
+edgedb server start
+===================
+
+Starts an EdgeDB server instance with the specified name.
+
+Synopsis
+--------
+
+``edgedb server start [options] [<name>]``
+
+Options
+-------
+
+``<name>``
+  The name of the EdgeDB instance.  Must be unique.  If not specified,
+  the name ``default`` is used.
+
+``--version=<ver>``
+  Optionally specifies the server version to use.  If not specified,
+  the latest installed server version is used.
+
+``--start-conf=auto|manual``
+  If set to ``auto`` (the default), the server will be started automatically
+  on system boot.
+
+``--port=<port-number>``
+  Optionally specifies the port number on which the server should listen.
+
+``--system``
+  By default, ``edgedb server start`` runs the server in the user scope,
+  if ``--system`` is specified, it is started as a system-wide service
+  instead.
+
+``--server-options -- <options>``
+  Passes ``edgedb-server`` options verbatim.  Must be the last argument.
+
+
+edgedb server status
+====================
+
+Shows the status of the specified server instance or all instances.
+
+Synopsis
+--------
+
+``edgedb server status [options] [<name>]``
+
+Options
+-------
+
+``<name>``
+  The name of the EdgeDB instance.  If not specified, the name
+  ``default`` is used.
+
+``--all``
+  Show the status of all known instances.
+
+Implementation
+--------------
+
+The command outputs the state of the server instance
+(``running`` or ``stopped``), the port number it is configured to run on,
+the scope of the instance (system-wide or user-local), and the runtime under
+which the server is running (docker or native).
+
+
+edgedb server stop
+==================
+
+Stops the specified EdgeDB server instance.
+
+Synopsis
+--------
+
+``edgedb server stop [options] [<name>]``
+
+Options
+-------
+
+``<name>``
+  The name of the EdgeDB instance.  If not specified, the name
+  ``default`` is used.
+
+
+edgedb server upgrade
+=====================
+
+Upgrades the specified EdgeDB server instance to a given EdgeDB version.
+
+Synopsis
+--------
+
+``edgedb server upgrade [options] [<name>]``
+
+Options
+-------
+
+``<name>``
+  The name of the EdgeDB instance.  If not specified, the name
+  ``default`` is used.
```

Comment:
If `<name>` is not specified the command should ask the user for confirmation. Probably same applies to all other commands.

Reply:
I think that the vast majority of users wouldn't actually *need* multiple instances at all, and so, to keep things simple, the instance name is optional.